### PR TITLE
Navbar: added variables / Page: added margin property

### PR DESCRIPTION
**Pull request by zemqi**
Please ensure your pull request adheres to the following guidelines:

- [ ] Use the following format: `* [owner/repo](link)`
- [ ] Link additions should be added to the bottom of the relevant category.
- [ ] New categories or improvements to the existing categorization are welcome.
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] Sort by alphabetical order

Thanks for contributing!

### CODE REVIEW

File: components/layouts/page/page.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 import { Container, MetaHead, PageHeading } from '../../'
 
 // Config
 import { Brand, Canonical } from 'config'
 
-export const Page = ({ children, fluid, meta, pageHeading }) => {
+export const Page = ({ children, fluid, marginTop, meta, pageHeading }) => {
   return (
-    <StyledPage>
+    <StyledPage marginTop={marginTop}>
```
Comment (iwarner) [inline components/layouts/page/page.js:20]: @zemqi I removed the margins from here in favour of using the Spacer later - also I would prefer not to use MarginTop - please remove for now.

File: components/molecules/navbar/components/brand/brand.js [hunk 2]
```diff
@@ -10,22 +10,50 @@ import { string } from 'prop-types'
 import { Image, Link } from '../../../../'
 
 // Style
-import styled from 'styled-components'
+import styled, { keyframes } from 'styled-components'
```
Comment (iwarner) [inline components/molecules/navbar/components/brand/brand.js:13]: @zemqi Will this work on all logos we use?

**Comment by iwarner** [comment]
@zemqi All changes should come with better stories to show the new properties etc. Please review the comments and ammend ASAP

File: components/layouts/page/page.js (hunk 1 above)
Comment (zemqi) [reply components/layouts/page/page.js:20]: How to up page and compare with transparent Navbar (for example), as I did win DryKISS to compare borders of logo and hero. Add negative margins to Navbar is better?

File: components/molecules/navbar/components/brand/brand.js (hunk 2 above)
Comment (zemqi) [reply components/molecules/navbar/components/brand/brand.js:13]: I saved standard logo and added wrapper to create logos like our DryKISS.
Also, maybe border style need to separate from component and put into drykiss.com? These places what I mean — need to refactor.

File: components/layouts/page/page.js (hunk 1 above)
Comment (iwarner) [reply components/layouts/page/page.js:20]: @zemqi Just remove for now I will solve this later, I have a better plan for the layouts

File: components/molecules/navbar/components/brand/brand.js (hunk 2 above)
Comment (iwarner) [reply components/molecules/navbar/components/brand/brand.js:13]: @zemqi Remember IUI is for global users so we need to decied whether this stuff belongs in IUI or specific to us... if us then we would put in parent app..

**Pull request merged**